spec/services/hashtag_autocomplete_service_spec.rb

```diff
@@ -167,19 +174,26 @@ def self.search(guardian_scoped, term, limit)
     context "when multiple tags and categories are returned" do
       fab!(:category2) { Fabricate(:category, name: "Book Zone", slug: "book-zone") }
       fab!(:category3) { Fabricate(:category, name: "Book Dome", slug: "book-dome") }
+      fab!(:category4) { Fabricate(:category, name: "Bookworld", slug: "book") }
       fab!(:tag2) { Fabricate(:tag, name: "mid-books") }
       fab!(:tag3) { Fabricate(:tag, name: "terrible-books") }
       fab!(:tag4) { Fabricate(:tag, name: "book") }
 
       it "orders them by name within their type order" do
         expect(subject.search("book", %w[category tag], limit: 10).map(&:ref)).to eq(
-          %w[book-club book-dome book-zone book great-books mid-books terrible-books],
+          %w[book book::tag book-club book-dome book-zone great-books mid-books terrible-books],
         )
       end
 
       it "orders correctly with lower limits" do
         expect(subject.search("book", %w[category tag], limit: 5).map(&:ref)).to eq(
-          %w[book-club book-dome book-zone book great-books],
+          %w[book book::tag book-club book-dome book-zone],
+        )
+      end
+
+      it "prioritises exact matches to the top of the list" do
```

Comment:
As said earlier, I don't see `exact_match` being used here even if the spec says "prioritises **exact matches** to the top of the list"